config.rdf

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="owl2html.xsl"?>
+<rdf:RDF 
+     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+     xmlns:owl="http://www.w3.org/2002/07/owl#"
+     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
+     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+
+    <owl:Ontology rdf:about="http://fedora.info/definitions/v4/config#">
+        <rdfs:label xml:lang="en">Fedora Commons Configuration Ontology</rdfs:label>
+        <rdfs:comment xml:lang="en">Ontology for user-defined configuration properties and classes on Fedora-curated resources.</rdfs:comment>
+    </owl:Ontology>
+
+    <owl:DatatypeProperty rdf:about="http://fedora.info/definitions/v4/config#hasIndexingTransformation">
+        <rdfs:label xml:lang="en">has indexing transformation</rdfs:label>
+        <rdfs:domain rdf:resource="http://fcrepo.info/definitions/v4/repository#Container"/>
```

Comment:
Good point-- I was mixing up the various levels of hierarchy available in Git/GitHub. I suppose I mean that we could either:

- Subdivide this repository with folders for "repository ontologies", "indexing ontologies", etc. This would have to evolve as the ontologies evolve with the HTTP API specs.
- Start a new "indexing SPI" repository and put the indexing ontology there. This is probably better, because there might be additional resources for that SPI, beyond the ontology.